ScoutBot/Database/Model/SheetAccess.cs

```diff
@@ -0,0 +1,20 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace ScoutBot.Database.Model
+{
+    /// <summary>
+    /// Used to track what discord roles can access which google sheets.
+    /// </summary>
+    public class SheetAccess
```

Comment:
You'll probably want a server ID here aswell, even if it's only one server for now.

Reply:
Why would I want a serverId on this table? I'm assuming this would be a foreign key in a different table down the line if I wanted to host this in different tables?

Reply:
Is RoleID universally unique across multiple servers?

Reply:
Yes. Its a "snowflake" object which is a unique id with a timestamp. 
![image](https://user-images.githubusercontent.com/35534232/94623900-1d855b80-026a-11eb-9ebe-c780757e683e.png)

Reply:
Due to that, I'm going to leave off the server identifier for now.